pulp_ansible/app/urls.py

```diff
@@ -10,9 +10,12 @@
     RoleVersionList
 )
 
+from pulp_ansible.app.viewsets import CollectionUploadViewSet
+
 
 urlpatterns = [
     url(r'pulp_ansible/galaxy/(?P<path>.+)/api/$', GalaxyVersionView.as_view()),
+    url(r'ansible/collection/$', CollectionUploadViewSet.as_view({'post': 'create'})),
```

Comment:
Why `collection` instead of `collections`?

Reply:
I think plural would be better. I'll switch it in next push.